Update firebase.json

pavan-ambekar opened the pull request.

pavan-ambekar committed:
Update firebase.json

pavan-ambekar commented:
Please update the firebase.json

UZAIR78920 commented:
Thank you sir it worked
Sir I have a doubt why did it work when src was replaced with dist??



On Tue, 17 Oct, 2023, 8:11 pm Pavan A, ***@***.***> wrote:

> Please update the firebase.json
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/UZAIR78920/pensieve/pull/1#issuecomment-1766560677>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ7QCRJP46L3KQCO625UU5LX72KIDAVCNFSM6AAAAAA6D7EFZ6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTONRWGU3DANRXG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

pavan-ambekar commented:
I'm sorry, I was stuck at work and didn't see your message.

Q. Why it worked when we changed `src` to `dist` in ` firebase.json`
A. when we run `npm run build` in the vite app it'll create a react **bundle** and that will be saved in `dist` directory

and we need to deploy the bundle that is the reason I changed and it worked

Many Thanks
